README.md

```diff
@@ -279,3 +280,59 @@ Q_MS_LCMS_METHODS - openBIS code from the vocabulary of LCMS methods
 technical_replicate - free text to denote replicates
 
 workflow_type - DDA or DIA
+
+
+### Imaging data with an OMERO server instance
+
+**Responsible dropbox:**
+[QBiC-register-omero-metadata](drop-boxes/register-omero-metadata)
+
+**Resulting data model in openBIS**  
+...Q_BIOLOGICAL_SAMPLE -> one Q_BMI_GENERIC_IMAGING_RUN per data file
```

Comment:
Can you elaborate? I do not get the meaning of this.

Reply:
For each tissue sample multiple images (the data files) can be created, so multiple Q_BMI_GENERIC_IMAGING_RUN samples are created and attached to that tissue sample

Reply:
```suggestion
For each tissue sample multiple images (the data files) can be created, so multiple Q_BMI_GENERIC_IMAGING_RUN samples are created and attached to that tissue sample
...Q_BIOLOGICAL_SAMPLE -> one Q_BMI_GENERIC_IMAGING_RUN per data file
```

Reply:
fine with me, if andreas agrees